src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -146,6 +146,7 @@ protected static function getBundleDefaultConfig()
                 'enabled' => false,
                 'fallbacks' => array('en'),
                 'logging' => true,
+                'cache' => 'translation.cache.default',
```

Comment:
We will just duplicate tests if we add these case, as it already covered by the container.